charts/gateway/production-values.yaml

```diff
@@ -528,10 +535,22 @@ otk:
     #   keyspace:
     # For Cassandra driverConfig is supported from GW 11.0. Either otk.database.cassandra.driverConfig or otk.database.properties should be provided
     #   driverConfig: {"localDataCenterName" : "DC1"}
+        # Optionally configure read only Data Base connection for mysql/oracle.
+    readOnlyConnection:
+      enabled: false
+      connectionName: OAuth_ReadOnly
+      # existingSecretName: otkReadOnlyDbSecret
+      # username: root
+      # password: 7layer
+      # properties: {"maximumPoolSize":15, "minimumPoolSize":3}
+      # jdbcURL: jdbc:mysql://<host>:<port>/<database>
+      # jdbcDriverClass: com.mysql.jdbc.Driver
+      # Oracle database name
+      # databaseName:
 
   # Install OTK Health check bundle on gateway. Uses config map.
   # Alternatively the bundle can be loaded using config map external to helm (useExisting: true)
-  # The bundle content can be found at https://github.com/CAAPIM/apim-charts/blob/stable/charts/gateway/bundles/otk-healthcheck.bundle
+  # The bundle content can be found at https://github.com/CAAPIM/apim-charts/blob/stable/charts/gateway/bundles/otkhealthcheck.bundle
```

Comment:
otkhealthcheck.bundle doesn't appear to exist

Reply:
Done